ivadomed/testing.py

```diff
@@ -182,6 +182,9 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
 
             if model_params["is_2d"]:
                 preds_idx_arr = None
+                idx_slice = batch['input_metadata'][smp_idx][0]['slice_index']
+                n_slices = batch['input_metadata'][smp_idx][0]['data_shape'][-1]
```

Comment:
@charleygros, before merging the PR, I would like to confirm with you that this will work for coronal and sagittal axis as well. Here I extract the number of slices based on the last dimension of the `data_shape`. I think it works because the dimension are swapped in orient_shapes_hwd [here](https://github.com/ivadomed/ivadomed/blob/cc16227f1fca40ec4619027563f1f2be513944f7/ivadomed/loader/loader.py#L262) according to the `slice_axis` param. However, I do not have any dataset/model to test it. Let me know your thoughts, thanks!

Reply:
Fully agree with your reasoning! Testing it now

Reply:
All green